python/tester/upload.py

```diff
@@ -390,6 +401,8 @@ def upload_from_pool(visit, snap_id):
             src_blob = snap_dict[snap_id][visit]
             exposure_num, headers = \
                 make_exposure_id(visit.instrument, visit.groupId, snap_id)
+            if instrument == "LSSTCam-imSim":
+                headers.update(make_imsim_time_headers(EXPOSURE_INTERVAL))
```

Comment:
I see that `make_imsim_time_headers` also calculates time.now(), which will be slightly later than `now` calculated a few lines up from here. Is that intentional, or should you pass in `now` to `make_imsim_time_headers`?